mlflow/metrics/metric_definitions.py

```diff
@@ -18,15 +23,15 @@ def standard_aggregations(scores):
 
 
 def _validate_text_data(data, metric_name, column_name):
```

Comment:
Sorry, one small nit. Can we rename `column_name` to `error_string` or something. Since its no more a column_name.

Reply:
changed wording to `col_specifier` for this and the target and prediction static messages as well.

I think it's more specific to the purpose of the string + reads better in the error text